Engine/TransactionHandlers/BrokerageTransactionHandler.cs

```diff
@@ -820,11 +818,16 @@ private void HandleOrderEvent(OrderEvent fill)
             {
                 Log.Debug("BrokerageTransactionHandler.HandleOrderEvent(): " + fill);
                 Interlocked.Exchange(ref _lastFillTimeTicks, DateTime.Now.Ticks);
+
+                // set the price currency on the fill and on the order
+                var security = _algorithm.Securities[fill.Symbol];
+                fill.FillPriceCurrency = security.SymbolProperties.QuoteCurrency;
+                order.PriceCurrency = fill.FillPriceCurrency;
```

Comment:
We also need to populate the PriceCurrency field in the Order class (needed for the serialization of trades). So should both fields be populated inside the brokerage fill events ?
Also, the OrderEvent constructor has an Order parameter, so maybe it's best to set the currency in the Order before passing it to the OrderEvent constructor ?
About the question of different currencies in brokerage vs. config, maybe log a warning and continue ?